tools/nnyeah/tests/utils/Compiler.cs

```diff
@@ -18,6 +18,7 @@ public enum PlatformName {
 
 	public class Compiler {
 		const string MonoCompiler = "/Library/Frameworks/Mono.framework/Versions/Current/Commands/csc";
+		const string buildDirectory = "../../../../../../_build";
```

Comment:
The Configuration class already contains all the logic you're adding:

https://github.com/xamarin/xamarin-macios/blob/92eda7f35399f442f8730aa539571ad8fd44d7da/tests/common/Configuration.cs#L14

It should be possible to just add that file to your project (although you might have to add more files too).

Other comments point to more specific locations in the Configuration class.

Reply:
This `buildDirectory` variable doesn't seem to be needed anymore:

```suggestion
```